### ISSUE TITLE
Please add suport for patch dwm-extrabar 20210930-a786211 version

### ISSUE THREAD

**Issue by a023e** [sipi/dwm-status2d#8]

**Comment by MokuKen** [comment]
Yes please add support for patch dwm-extrabar 20210930-a786211 version 

when i tried to combine them only top is working have collors but in the 2 statuses in bottom show the color numbers not changing to colors

**Comment by saifshahriar** [comment]
I was able make the bottom right corner work. But couldn't figure out the bottom left corner yet.

**Comment by saifshahriar** [comment]
**Update:** Patched it. 😊
I will be making a pull request, if you don't mind. Feel free to change it as you wish.

**Comment by sipi** [comment]
Thanks a lot,
Sorry for my lack of follow-up.

**Comment by saifshahriar** [comment]
@sipi Previous patch for dwm 6.4 works perfectly fine for dwm 6.5.

![Image](https://github.com/user-attachments/assets/d03f107c-a7ab-46e4-b775-8c975c7e197b)
